Known_Issues/Windows_Health_Info.md

```diff
@@ -0,0 +1,18 @@
+## Issue ##
+When trying to use Azure Serial Console to connect to a Windows virtual machine you receive VM health information similar to the example below, but the VM does not respond to pressing enter or other input.
+
+```
+======== Microsoft Azure VM Health Report - Start 2018-03-06T02:06:40.6256349Z ========
```

Comment:
should we remove any specific details (eg. macAddress/ipv6 address)

Reply:
Ok, I modified them prior but will make it more explicit